test/views/wallet/TransactionView/__snapshots__/TransactionView.test.tsx.snap

```diff
@@ -178,6 +178,7 @@ exports[`TransactionView component render TransactionView renders correctly 1`]
                             >
                               <span
                                 class="MuiBox-root MuiBox-root"
+                                style="color: rgb(255, 255, 255);"
```

Comment:
Shouldn't we be taking colors from a file?

Reply:
Yeah, I need to go back and clean a few things up, I was just getting the component implemented quickly. Want to set the theme stuff up correctly.

Reply:
snapshot!